### PR TITLE
Fix build on Windows and bug in base quaternion initialisation

### PR DESCRIPTION
**Pull request by lucaTagliapietra**
Hi guys, 
please review and merge this pull request.

Main changes:
- add _USE_MATH_DEFINES and move math include on top of the file to fix build on Windows.
- fix the bug in the initialisation of the base orientation quaternion which caused rviz to rise errors about the quaternion format when keeping the default base orientation.

As side comment, I think we can keep as good practice placing on the top the _USE_MATH_DEFINE definition and cmath inclusion also for the other files and other projects which needs to be multiplatform (aka build also on Windows).  @francesco-romano c.c. @diegoferigo  @traversaro

### CODE REVIEW

**Comment by traversaro** [comment]
This PR is good to go, but FYI nn alternative to move the `cmath` to be the first included file of compilation unit is to specify the `_USE_MATH_DEFINES` in CMake, see https://github.com/robotology/QA/issues/81 .

**Comment by lucaTagliapietra** [comment]
I know, but, generally speaking, I don't like that much touching the CMake for stuff like this one unless there are no other ways or a specific reason to do that.
However we can discuss about it.

**Comment by traversaro** [comment]
No problem, both solutions are equally fine for me (until we are discussing in stuff defined in the headers).

**Comment by francesco-romano** [comment]
Honestly I prefer adding the macro to the CMake instead of adding it in all the header files.

Anyway, the repo maintainers are now @lucaTagliapietra @diegoferigo and @claudia-lat , so it is their decision

**Comment by diegoferigo** [comment]
If there are no issues for importing targets in external projects (if needed in the future), I'm ok with the CMake way. It is cleaner, we avoid to do it manually every time in the source, and the order of the inclusion of the headers becomes independent hence less error prone.

**Comment by traversaro** [comment]
@diegoferigo If you think the CMake handling of the `_USE_MATH_DEFINES ` macro is blocking this PR from being merged, I suggest to modify your review to "Ask for Changes".

**Comment by traversaro** [comment]
@lucaTagliapietra @diegoferigo  can we merge this PR?

**Comment by francesco-romano** [comment]
@traversaro I would still prefer the CMake way, instead of polluting each .cpp file with the define

**Comment by traversaro** [comment]
No problem for me, I was just doing the periodic ping of stale PRs.

**Comment by diegoferigo** [comment]
I remember I pushed https://github.com/lucaTagliapietra/human-dynamics-estimation/pull/1 some while ago, we'll handle this when @lucaTagliapietra will join us again

**Comment by lucaTagliapietra** [comment]
Fine for me the CMake way.
I was (and I still am) waiting for @claudia-lat  coming back to test it again on the laptop she has with her. 
Next Monday will probably be the day ;)

**Comment by francesco-romano** [comment]
I did some modifications, but I do not have access to the PR. 
Can you enable the push permissions to this PR?

**Comment by lucaTagliapietra** [comment]
@francesco-romano you should have write permission now. Could you please check?

**Comment by francesco-romano** [comment]
just pushed.

Anyway, (for the future), usually there is a tick in the PR to allow people to push on your branch without giving explicit permissions

**Comment by lucaTagliapietra** [comment]
Yeap, I know. I've probably forgot to check it.
I'll test the mods on Monday on lab laptop, then we can merge and close.

File: human-viz-bridge/CMakeLists.txt [hunk 1]
```diff
@@ -34,6 +38,10 @@ list(APPEND conf config/human-tf-bridge.ini)
                                                 
 add_executable(human-jointstate-bridge src/HumanJointStateBridge.cpp)
 
+if (MSVC)
+  target_compile_definitions(human-jointstate-bridgePRIVATE -D_USE_MATH_DEFINES)
```
Comment (traversaro) [inline human-viz-bridge/CMakeLists.txt]: Missing space?

**Pull request merged**